packages/graphql/src/schema/query.ts

```diff
@@ -291,7 +317,8 @@ export const Query = builder.queryType({
           return QuestionService.getSingleQuestion(args, ctx)
         },
       }),
-      singleMicroSession: asAuthenticated.field({
+
+      singleMicroSession: asUser.field({
```

Comment:
Changed from `asAuthenticated` to `asUser` as this query is only used in `frontend-manage`